source/reference/mongosync-behavior.txt

```diff
@@ -296,12 +154,43 @@ Read Preference
 connecting to the source and destination clusters. For more information,
 see :ref:`connections-read-preference`. 
 
-.. _c2c-capped-collections:
+Destination Cluster Considerations
+----------------------------------
```

Comment:
Since all of these sections are considerations (ie, Cluster and Collection Type Considerations, Performance Considerations, etc) I think we can remove "Considerations" here.
```suggestion
Destination Clusters
--------------------
```

Reply:
Makes sense, making change now!